src/components/LeftPanel/LeftPanel.js

```diff
@@ -35,13 +38,30 @@ class LeftPanel extends React.Component {
     return panel === this.props.activePanel ? 'flex' : 'none';
   };
 
+  onWidthChange = width => {
+    if (isIE11 || isIEEdge) {
+      this.props.updateLeftPanel(width);
+    }
+  }
+
   render() {
-    const { isDisabled, closeElement, customPanels } = this.props;
-    
+    const { isDisabled, closeElement, customPanels, leftPanelWidth } = this.props;
     if (isDisabled) {
       return null;
     }
-    
+
+    let style = { };
+    let innerStyle = { };
+
+    if (isIE11) {
+      // IE11 will use javascript for controlling width, other broswers will use CSS
+      style = leftPanelWidth ? { width: leftPanelWidth } : null;
+
+      innerStyle = {
+        left: leftPanelWidth
```

Comment:
I don't remember the reason why we are using `left` for absolutely positioning the resize bar.

In `LeftPanel.scss`, we could use `right: 0px` and `margin-right: -10px` and probably get rid of this.

Reply:
Using CSS for this is better. Before it was using CSS variable for "left" but since that wasn't supported in IE11, I made it set "left" with JavaScript instead. I tested with `right: 0px` and it seem to work in IE